cors.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2021-2023 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package connect
+
+import (
+	"net/http"
+)
+
+// CORS config to help implement Cross-Origin Resource Sharing (CORS) header support.
```

Comment:
Can we integrate this a little more closely with the HTML docs? Maybe we should include a link to the docs right now, and plan to update the docs to use this code once it's included in a release.